CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+  - [561](https://github.com/thoth-pub/thoth/issues/561) - Add PUBLISHER_WEBSITE as a location platform
```

Comment:
Underscores should be escaped in markdown (or alternatively you can just use the human-readable name)